Adyen/Model/Payment.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Describes the current payment.
+public struct Payment {
```

Comment:
can `amount` and `country` code be constants?

Reply:
👍